examples_utils/benchmarks/run_benchmarks.py

```diff
@@ -479,9 +498,16 @@ def run_benchmarks(args: argparse.Namespace):
             with
 
     """
+
+    spec_files = ",".join([str(sf) for sf in args.spec if ".yml" in str(sf)])
+    logger.info(f"Running benchmark suite: '{spec_files}'")
+
+    # Load all benchmark configs from all files given
+    spec = {}
+    for spec_file in args.spec:
+        logger.info(f"Examining: '{spec_file}'")
```

Comment:
These two bits of logging seem quite similar, did you mean to have both?

Reply:
Yh true, redundant info. I'll remove the first one